playbooks/run.yaml

```
@@ -0,0 +1,17 @@
+- name: Setup main server at hetzner
+  hosts: ci_server
+  pre_tasks:
+    - name: Collect facts
+      ansible.builtin.setup:
+        gather_subset:
+          - 'system'
+          - 'local'
+          - 'distribution'
+  roles:
+    - role: common
+    - role: docker
+    - role: caddy
+    - role: woodpecker_server
+    - role: woodpecker_agent
```

```suggestion
    # - role: woodpecker_agent
```

We only use the autoscaler atm as the server is the smallest arm machine.